kubernetes/cray-sysmgmt-health/.gitignore

```diff
@@ -0,0 +1,2 @@
+charts/
+requirements.lock
```

Comment:
since requirements.lock is removed, this line can also be removed from the file.

Reply:
The requirements.lock file is added if a developer does `helm dep up`. It's left here because I don't want a developer to accidentally add this file. It's happened often in the past.